src/main/resources/assets/spruceui/lang/it_it.json

```diff
@@ -1,11 +1,12 @@
 {
-  "spruceui.reset": "Ripristina",
-  "spruceui.gui.unbind": "Disassocia",
-  "spruceui.narrator.separator": "Separatore %s",
+  "spruceui.reset": "Resetta",
+  "spruceui.gui.unbind": "Disattiva",
+  "spruceui.narrator.separator": "%s separatore",
+  "spruceui.narration.toggle_switch": "Disattiva interruttore: %s, con stato %s",
   "spruceui.options.generic": "%s: %s",
-  "spruceui.options.generic.fancy": "Sofisticato",
-  "spruceui.options.generic.fast": "Semplice",
-  "spruceui.options.generic.fastest": "Semplicissimo",
+  "spruceui.options.generic.fancy": "Fancy",
+  "spruceui.options.generic.fast": "Veloce",
+  "spruceui.options.generic.fastest": "Velocissimo",
```

Comment:
While veloce and velocissimo can be good changes, they are consistent with the vanilla language and I believe it's less confusing for people to have them all consistent.
Translating sofisticato to fancy is just a loss of information.